documentation/manual/views.md

```diff
@@ -54,12 +54,12 @@ Note that there exist [cases](https://github.com/GIScience/oshdb/issues/87) wher
 GroupByEntity
 -------------
 
-The [`groupByEntity()`](https://docs.ohsome.org/java/oshdb/0.5.9/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#groupByEntity--) method of a MapReducer slightly changes the way the MapReducers recieves and transforms values: Instead of iterating over each snapshot or contribution individually, in this mode all snapshots or all contributinos of an individual OSM entity are collected into a list of values first. This makes it possible to investigate the full edit history of individual OSM objects at once.
+The [`groupByEntity()`](https://docs.ohsome.org/java/oshdb/0.5.9/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#groupByEntity--) method of a MapReducer slightly changes the way the MapReducers recieves and transforms values: Instead of iterating over each snapshot or contribution individually, in this mode all snapshots or all contributions of an individual OSM entity are collected into a (by timestamp sorted) list first. This makes it possible to investigate the full edit history of individual OSM objects at once, which is for example needed when one is looking for contributions that got reverted at a later point in time.
```

Comment:
the links are updated semi-automatically (see step 4 in https://confluence.gistools.geog.uni-heidelberg.de/display/oshdb/how+to+create+a+release): since the creation of a release requires quite some manual steps anyway, I think updating these links as well shouldn't be a big deal. With explicit version links, you have the (small) benefit that even if people look would look at old versions of the documentation, the links would still point to the correct javadoc pages, while a `/…/latest|stable/…` link might break if at some point a class or package gets renamed (e.g. in #241).